frontend/src/pages/files/FileControlsWrapper.tsx

```diff
@@ -165,13 +165,13 @@ const FileManager = observer((props: FileManagerProps) => {
         securityNotice = (
           <>
             <h4>
-              <span className="badge rounded-pill p-1 px-2 bg-success mb-2">PUBLISHED SUBFOLDER</span>
+              <span className="badge rounded-pill p-1 px-2 bg-success mb-2">FINISHED SUBFOLDER</span>
```

Comment:
Did you want to switch to "Finalized" (based on your comment on #87). I'm fine either way.

Reply:
You are right, I'll fix it as soon as I can.